hybridse/src/udf/default_udf_library.cc

```diff
@@ -1631,7 +1631,9 @@ void DefaultUdfLibrary::InitMathUdf() {
                 SELECT RADIANS(90.0);
                 --output 1.570796326794896619231
             @endcode
-            @since 0.6.0)");
+
+            @since 0.6.0)")
```

Comment:
new added types are not supported in `0.6.0`. update the since ?

Reply:
no, I fix this as bug, so version number not change. If changed all signatures are updated to the new version.